package.json

```
@@ -17,7 +17,7 @@
   ],
   "author": "",
   "license": "ISC",
-  "dependencies": {
+  "dev-dependencies": {
```

```suggestion
  "devDependencies": {
```

Good point, it should be a dev dependency. Should be camelCased but happy to accept otherwise :D